.github/workflows/linkchecker.yml

```
@@ -0,0 +1,33 @@
+name: Link checks
+
+on:
+  # Run manually by clicking a button in the UI
+  workflow_dispatch:
+  # Run every 8 hours
+  schedule:
+    - cron: '0 */8 * * *'
```

Maybe 24 will be enough?

---

sure, I'm ok with that. This will run once a day at 8:00a UTC

```suggestion
    - cron: '0 8 * * *'
```